app/controllers/orders_controller.rb

```diff
@@ -14,11 +14,8 @@ def show
 
   # GET /orders/new
   def new
-    if current_dispatcher.present?
+      @dispatcher = Dispatcher.find(current_dispatcher.id)
```

Comment:
Clients can create new orders also. Remember this. It's better to use `||=` instead of `=` there.
You can ask @SudakovEugene and @yelizaveta-dziuba about this.